filemanager/widgets.py

```diff
@@ -37,12 +37,12 @@ class Media:
 
     def render(self, name, value, attrs=None):
         rendered = super(CKEditorWidget, self).render(name, value, attrs)
-        div = "<div style='height:20px'></div>"
+        div = '<div style='height:20px'></div>'
```

Comment:
Are you sure this will work @OscarEReyes

Reply:
Oh, I see what I did there.
My bad, I placed single quotes inside single quotes. 
I corrected that,  do i need to make a new pull request